crypto/ec/ec_backend.c

```diff
@@ -10,15 +10,177 @@
 #include <openssl/core_names.h>
 #include <openssl/objects.h>
 #include <openssl/params.h>
+#include <openssl/err.h>
 #include "crypto/bn.h"
 #include "crypto/ec.h"
+#include "ec_local.h"
+#include "e_os.h"
+#include "internal/param_build_set.h"
+
+/* Mapping between a flag and a name */
+static const OSSL_ITEM encoding_nameid_map[] = {
+    { OPENSSL_EC_EXPLICIT_CURVE, OSSL_PKEY_EC_ENCODING_EXPLICIT },
+    { OPENSSL_EC_NAMED_CURVE, OSSL_PKEY_EC_ENCODING_GROUP },
+};
+
+int ec_encoding_name2id(const char *name)
+{
+    size_t i, sz;
+
+    if (name == NULL)
+        return OPENSSL_EC_NAMED_CURVE;
```

Comment:
???????
No name => named curve?

Reply:
It will select the default value if NULL is passed.

Reply:
That's not at all obvious. Mind adding a comment explaining this?

Reply:
Done